modules/ROOT/pages/policy-openid-connect.adoc

```diff
@@ -36,12 +36,12 @@ WARNING: For performing the OAuth dance, you will be required to provide client
 
 == Performance
 
-To improve performance, once a token is validated, by default, the token is cached by the system until it expires. Sometimes it is desirable to reduce or even disable the caching mechanism, for example to take into account revoked token scenarios.
+To improve performance, once a token is validated, it is cached by the system by default, until the token expires. In certain cases, you might reduce or even disable the caching mechanism, for example to take into account revoked token scenarios.
 
-To control token cached time (ie token cache time before retrying validation against the OAuth Provider), you may specify the following property:
+To control token cached time before retrying validation against the OAuth Provider), specify the following property:
 
 [listing]
-anypoint.platform.max_federation_expiration_time=<a number equal or greater than 0>
+anypoint.platform.max_federation_expiration_time=<numeric value indicating time in seconds>
```

Comment:
Should there be a sentence saying something like "To disable caching, set the value to zero." ?

Reply:
Done! Thanks!